client-v2/src/components/FrontsEdit/Front.tsx

```diff
@@ -236,23 +288,30 @@ class FrontComponent extends React.Component<FrontProps, FrontState> {
                 />
               </OverviewToggleContainer>
             </SectionContentMetaContainer>
-            <FrontCollectionsContainer>
+            <FrontCollectionsContainer
+              onScroll={this.handleScroll}
+              innerRef={ref => (this.collectionContainerRef = ref)}
+            >
               <Root id={this.props.id} data-testid={this.props.id}>
                 {front.collections.map(collectionId => (
-                  <Collection
-                    key={collectionId}
-                    id={collectionId}
-                    frontId={this.props.id}
-                    priority={front.priority}
-                    browsingStage={this.props.browsingStage}
-                    alsoOn={this.props.alsoOn}
-                    handleInsert={this.handleInsert}
-                    handleMove={this.handleMove}
-                    selectArticleFragment={this.props.selectArticleFragment(
-                      this.props.id
-                    )}
-                    handleArticleFocus={this.handleArticleFocus}
-                  />
+                  <CollectionContainer
+                    innerRef={ref => (this.collectionRefs[collectionId] = ref)}
```

Comment:
Quick thing - can we rename `ref` here to something like `element`? just get a bit confused between the param and the attribute `ref`.

Reply:
Great point 👍